lib/std/fmt/parse_hex_float.zig

```diff
@@ -207,16 +207,19 @@ pub fn parseHexFloat(comptime T: type, s: []const u8) !T {
     // - We've truncated exactly 0.5ULP (R=1 S=0), increase the mantissa if the
     //   result is odd (G=1).
     // The two checks can be neatly folded as follows.
-    mantissa |= @boolToInt(mantissa & 0b100 != 0);
-    mantissa += 1;
-
+    var round: u128 = 0;
+    round = @boolToInt(mantissa & 0b011 != 0);
     mantissa >>= 2;
+    round |= @boolToInt(mantissa & 0b001 != 0);
```

Comment:
`u1 | u1` will never be `0b11`, here's an accurate translation:
```suggestion
    var round = @truncate(u2, mantissa);
    mantissa >>= 2;
    round |= @truncate(u1, mantissa); 
```